test/index.js

```diff
@@ -264,42 +264,36 @@ describe('PLACEHOLDER', () => {
     ].join(''))
   })
 
-  // TODO: previously there was a generated whitespace after >
-  // TODO: height in the middle of those rules doesn't get grouped with other declaratations
-  // TODO: also some other things wrong with the stringified output here, need further investigation when previous points get fixed
-  // test('@media specifity', () => {
-  //  expect(stylis(`
-  //    > #box-not-working {
-  //      background:red;
-  //      padding-left:8px;
-  //      width:10px;
-  //      @media only screen and (min-width:10px) {
-  //        width:calc(
-  //          10px + 90px *
-  //          (100vw - 10px) / 90
-  //        );
-  //      }
-  //      @media only screen and (min-width:90px) {
-  //        width:90px;
-  //      }
-  //      height: 10px;
-  //      @media only screen and (min-width:10px) {
-  //        height:calc(
-  //          10px + 90px *
-  //          (100vw - 10px) / 90
-  //        );
-  //      }
-  //      @media only screen and (min-width:90px) {
-  //        height: 90px;
-  //      }
-  //    }`)).to.equal([
-  //      '.user > #box-not-working{background:red;padding-left:8px;width:10px;height:10px;}',
-  //      '@media only screen and (min-width:10px){.user > #box-not-working{width:calc( 10px + 90px * (100vw - 10px) / 90 );}}',
-  //      '@media only screen and (min-width:90px){.user > #box-not-working{width:90px;}}',
-  //      '@media only screen and (min-width:10px){.user > #box-not-working{height:calc( 10px + 90px * (100vw - 10px) / 90 );}}',
-  //      '@media only screen and (min-width:90px){.user > #box-not-working{height:90px;}}'
-  //    ].join(''))
-  // })
+  test('@media specifity', () => {
+    expect(
+      stylis(`
+        > #box-not-working {
+          background:red;
+          padding-left:8px;
+          width:10px;
+          @media only screen and (min-width:10px) {
+            width: calc(10px + 90px * (100vw - 10px) / 90);
```

Comment:
I've decided to move all calcs' here to single lines as this test isn't really concerned about multi-lines values and writing expected result for those is more cumbersome